src/View/IRenderable.php

```diff
@@ -0,0 +1,17 @@
+<?php
+declare(strict_types=1);
+
+namespace YapepBase\View;
+
+interface IRenderable
+{
+    /**
+     * Renders the object and sends the content to output.
+     */
+    public function render(): void;
+
+    /**
+     * Renders the object and returns the content
+     */
+    public function __toString(): string;
```

Comment:
Do we really need the magic version? Can we just have a regular non magic toString() method?

Reply:
What would be the benefit of not using the __toString ?